app/app.rb

```diff
@@ -0,0 +1,22 @@
+require 'sinatra/base'
+require_relative './data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  set :views, proc {File.join(root,'..','/app/views')}
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new_peep'
+  end
+
+  post '/peeps' do
+  Peep.create(message: params[:message])
+  redirect '/peeps'
+end
```

Comment:
Lint/BlockAlignment: `end` at 19, 0 is not aligned with `post '/peeps' do` at 16, 2